cmd/jaeger/internal/extension/jaegerstorage/extension.go

```diff
@@ -17,9 +17,9 @@ import (
 	"github.com/jaegertracing/jaeger/internal/storage/v1"
 	"github.com/jaegertracing/jaeger/internal/storage/v1/badger"
 	"github.com/jaegertracing/jaeger/internal/storage/v1/cassandra"
-	es "github.com/jaegertracing/jaeger/internal/storage/v1/elasticsearch"
 	"github.com/jaegertracing/jaeger/internal/storage/v1/memory"
 	"github.com/jaegertracing/jaeger/internal/storage/v2/api/tracestore"
+	es "github.com/jaegertracing/jaeger/internal/storage/v2/elasticsearch"
```

Comment:
nit: now that this overall PR is finished, I suggest we remove this file from it and make a follow-up PR, because it's a major functionality change to enable v2 ES storage, so I'd rather have a small dedicated PR for that.